src/tla/modules/box.tla

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright 2010-2025, Tarantool AUTHORS, please see AUTHORS file.
+ *
+ * Redistribution and use in source and binary forms, with or
+ * without modification, are permitted provided that the following
+ * conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above
+ *    copyright notice, this list of conditions and the
+ *    following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above
+ *    copyright notice, this list of conditions and the following
+ *    disclaimer in the documentation and/or other materials
+ *    provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY <COPYRIGHT HOLDER> ``AS IS'' AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED
+ * TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL
+ * <COPYRIGHT HOLDER> OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+ * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR
+ * BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF
+ * THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
+ * SUCH DAMAGE.
+ */
+
+--------------------------------- MODULE box -----------------------------------
+
+EXTENDS Integers, Sequences, FiniteSets
+
+--------------------------------------------------------------------------------
+\* Declaration
+--------------------------------------------------------------------------------
+
+CONSTANTS
+    Servers
+
+ASSUME Cardinality(Servers) > 0
+
+VARIABLES
+    error,          \* Critical error on the instance (only SplitBrain now).
+    vclock,         \* Vclock of the current instance.
+    txQueue,        \* Queue from any thread (except applier) to TX.
+    txApplierQueue  \* Queue from applier thread to Tx. Applier needs separate
+                    \* one, since it's crucial, that the write of synchro
+                    \* request is synchronous and none of the new entries
+                    \* are processed until this write is completed.
+
+boxVars == <<error, vclock, txQueue, txApplierQueue>>
+
+\* Applier substitution
+CONSTANTS SplitBrainCheck
+VARIABLES msgs, applierAckMsg, applierVclock
+LOCAL applierVarsSub == <<msgs, applierAckMsg, applierVclock>>
+
+\* Txn substitution.
+CONSTANTS MaxClientRequests
+VARIABLES tId, clientCtr, walQueue
+LOCAL txnVarsSub == <<tId, clientCtr, walQueue>>
+
+\* Raft substitution.
+CONSTANTS ElectionQuorum, MaxRaftTerm
+VARIABLES state, term, volatileTerm, vote, volatileVote, leader, votesReceived,
+          leaderWitnessMap, isBroadcastScheduled, candidateVclock,
+          relayRaftMsg
```

Comment:
I would place var per line, like you make above. It will make patches for specs mode readable.
Here and below.

Reply:
Now it's way better, I don't have so much variables. Do we still want to add them line by line? (same reason here, aesthetic)